utils/loggers/mlflow/mlflow_utils.py

```diff
@@ -56,6 +56,7 @@ def setup(self, opt: Namespace) -> None:
         self.client = mlflow.tracking.MlflowClient()
         self.log_params(vars(opt))
         self.log_metrics(vars(opt), is_param=True)
+        self.current_epoch = {}
```

Comment:
Should be set to 0? 
and might conflict on resumes - let me play around with this over the weekend, think there's different ways to handle resumes already